Accessibility statement updated (#5271)
DFE-Digital/register-trainee-teachers: app/views/pages/accessibility.md

  - navigate most of the service using just a keyboard
  - navigate most of the service using speech recognition software
  - listen to most of the content on the service using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)

We’ve also made the text on the service as simple as possible to understand.
<a href='https://mcmw.abilitynet.org.uk/' class='govuk-link' rel='noreferrer noopener' target="_blank">AbilityNet (opens in new tab)</a> has advice on making your device easier to use if you have a disability.

## How accessible this service is

We know some parts of this website are not fully accessible:

  - some of our forms do not provide suggestions on how to rectify information that has been input incorrectly
  - unsupported ARIA (Accessible Rich Internet Applications) attributes are used on some form inputs
  - error messages are announced twice to users of JAWS (Job Access With Speech)
  - files cannot be uploaded whilst navigating with speech recognition software

## Feedback and contact information

If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille: 

>   - email <a class='govuk-link' href='mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues%20'>becomingateacher@digital.education.gov.uk</a>

Reviewer: Do you need the HTML here or could you jus use `<becomingateacher@digital.education.gov.uk>`?

```suggestion
  - email <a class="govuk-link" href="mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues">becomingateacher@digital.education.gov.uk</a>
```

Reviewer 2: Do we actually need a single itemed list?

![image](https://github.com/user-attachments/assets/3377c06c-eba2-46bd-8691-e7fb13971d86)

We’ll consider your request and get back to you in 5 days.

## Reporting accessibility problems with Register trainee teachers

> We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact: <a class='govuk-link' href='mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues%20'>becomingateacher@digital.education.gov.uk</a>. Include ‘Accessibility’ in the subject line of your email.

Reviewer: ```suggestion
We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact: <a class="govuk-link" href="mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues">becomingateacher@digital.education.gov.uk</a>. Include ‘Accessibility issues’ in the subject line of your email.
```

## Enforcement procedure

> The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, <a href='https://www.equalityadvisoryservice.com/' class='govuk-link' rel='noreferrer noopener' target="_blank">contact the Equality Advisory and Support Service (opens in a new tab)</a>.

Reviewer: ```suggestion

The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, <a href="https://www.equalityadvisoryservice.com/" class="govuk-link" rel="noreferrer noopener" target="_blank">contact the Equality Advisory and Support Service (opens in a new tab)</a>.
```

## Technical information about this service’s accessibility

The Department for Education is committed to making its websites accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.

## Compliance status

This website is partially compliant with the WCAG (Web Content Accessibility Guidelines) version 2.2 AA standard, due to the following non-compliances (and/or) the following exemptions.

## Non-accessible content

The content is non-accessible for the following reasons:

### Specific error suggestions are not provided for some fields on forms

> This can be seen on date fields, if the user inputs an incorrectly formatted date, the error message does not provide a suggestion on how to rectify it. This fails WCAG 2.2 success criterion 3.3.1 Error Identification.

Reviewer: ```suggestion

This can be seen on date fields, if the user inputs an incorrectly formatted date, the error message does not provide a suggestion on how to rectify it. This fails WCAG 2.2 success criterion 3.3.1 Error Identification.
```

We’re waiting on third parties to assist with remediation so a fix date cannot be provided.

### Unsupported ARIA attributes are present on some form inputs

> This may cause issues for users of assistive technology. This fails WCAG 2.2 success criterion 4.1.2 Name, Role, Value.

Reviewer: ```suggestion

This may cause issues for users of assistive technology. This fails WCAG 2.2 success criterion 4.1.2 Name, Role, Value.
```

We’re waiting on third parties to assist with remediation so a fix date cannot be provided.

### While navigating with JAWS, parts of error messages are announced multiple times

> This fails WCAG 2.2 success criterion 4.1.2 Name. Role, Value.

Reviewer: ```suggestion

This fails WCAG 2.2 success criterion 4.1.2 Name. Role, Value.
```

We’re waiting on third parties to assist with remediation so a fix date cannot be provided.

## Disproportionate burden

We have determined that there is a disproportionate burden to meet all the requirements for the following reasons:

### Interactive tools

The upload file function is unavailable whilst navigating using speech recognition software. This is a known flaw of speech recognition software.

> We’ve assessed the cost of fixing the navigation issues and accessing information, as well as the costs associated with interactive tools and transactions. We believe that proceeding with these changes now would place a <a class='govuk-link' href="https://www.legislation.gov.uk/uksi/2018/952/regulation/7/made" rel="noreferrer noopener" target="_blank">disproportionate burden (opens in new tab)</a> within the meaning of the accessibility regulations. We will conduct another assessment when the supplier contract is up for renewal, likely in 2026.

Reviewer: ```suggestion
We’ve assessed the cost of fixing the navigation issues and accessing information, and the costs associated with interactive tools and transactions. We believe that proceeding with these changes now would place a <a class="govuk-link" href="https://www.legislation.gov.uk/uksi/2018/952/regulation/7/made" rel="noreferrer noopener" target="_blank">disproportionate burden (opens in new tab)</a> within the meaning of the accessibility regulations. We will conduct another assessment when the supplier contract is up for renewal, likely in 2026.
```

## What we’re doing to improve accessibility

> We have a backlog of tickets created for ongoing accessibility improvements. We conduct regular user testing and audits. We will continue to use the accessibility manual to enhance knowledge within the team and resolve accessibility issues when identified.

Reviewer: ```suggestion

We have created a backlog of tickets for ongoing accessibility improvements. We conduct regular user testing and audits. We will continue to use the accessibility manual to enhance knowledge within the team and resolve accessibility issues when identified.
```

## Preparation of this accessibility statement

This statement was prepared on 29 February 2024. It was last reviewed on 7 May 2025. 

This website was last tested on 29 February 2024. The test was carried out by Zoonou.

> Zoonou used <a class='govuk-link' href="https://www.w3.org/TR/WCAG-EM" rel='noreferrer noopener' target="_blank">WCAG-EM</a> to define the pages tested and test approach.

Reviewer: ```suggestion
Zoonou used <a class="govuk-link" href="https://www.w3.org/TR/WCAG-EM" rel="noreferrer noopener" target="_blank">WCAG-EM (opens in new tab)</a> to define the pages tested and test approach.
```

  - navigate most of the service using just a keyboard
  - navigate most of the service using speech recognition software
  - listen to most of the content on the service using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)

We’ve also made the text on the service as simple as possible to understand.
<a href='https://mcmw.abilitynet.org.uk/' class='govuk-link' rel='noreferrer noopener' target="_blank">AbilityNet (opens in new tab)</a> has advice on making your device easier to use if you have a disability.

## How accessible this service is

We know some parts of this website are not fully accessible:

  - some of our forms do not provide suggestions on how to rectify information that has been input incorrectly
  - unsupported ARIA (Accessible Rich Internet Applications) attributes are used on some form inputs
  - error messages are announced twice to users of JAWS (Job Access With Speech)
  - files cannot be uploaded whilst navigating with speech recognition software

## Feedback and contact information

If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille: 

>   - email <a class="govuk-link" href="mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues">becomingateacher@digital.education.gov.uk</a>

Reviewer 2: Do we actually need a single itemtised list?

![image](https://github.com/user-attachments/assets/807f75e3-45bc-457e-985e-7fc8eba6f019)

Author: It's formatted as a list in the accessibility template @defong . I can change it back to a paragraph.

title: Accessibility statement for Register trainee teachers
---
<p class='govuk-body'>This accessibility statement applies to Register trainee teachers.</p>

> <p class='govuk-body'>This website is run by the Department for Education. We want as many people as possible to be able to use this website. For example, that means you should be able to:</p>

Reviewer 2: Do we need to spell out the url for out website?

Contrasting against https://design-system.service.gov.uk/accessibility-statement/

Reviewer: Agreed. "This website" is ambiguous. Should the first instance say "Register trainee teachers"?

Author: I don't disagree, but again, it's the [accessibility template wording](https://accessibility.education.gov.uk/audits-statements/accessibility-statements/partially-compliant).

<ul class='govuk-list govuk-list--bullet'>
  <li>change colours, contrast levels and fonts</li>
  <li>zoom in up to 400% without the text spilling off the screen</li>
  <li>navigate most of the service using just a keyboard</li>
  <li>navigate most of the service using speech recognition software</li>
  <li>listen to most of the content on the service using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
</ul>
<p class='govuk-body'>We’ve also made the text on the service as simple as possible to understand.</p>

> <p class='govuk-body'><a href='https://mcmw.abilitynet.org.uk/' class='govuk-link' target="_blank">AbilityNet (opens in a new tab)</a> has advice on making your device easier to use if you have a disability.</p>

Reviewer: Add the `rel="noreferrer noopener"` as suggested by the [design system](https://design-system.service.gov.uk/styles/links/#external-links).

```suggestion
<p class='govuk-body'><a href='https://mcmw.abilitynet.org.uk/' class='govuk-link'  rel='noreferrer noopener' target="_blank">AbilityNet (opens in new tab)</a> has advice on making your device easier to use if you have a disability.</p>
```

<h3 class='govuk-heading-s'>Unsupported ARIA attributes are present on some form inputs</h3> 
<p class='govuk-body'>This may cause issues for users of assistive technology. This fails WCAG 2.2 success criterion 4.1.2 Name, Role, Value.</p>

<p class='govuk-body'>We’re waiting on third parties to assist with remediation so a fix date cannot be provided.</p>

<h3 class='govuk-heading-s'>While navigating with JAWS, parts of error messages are announced multiple times</h3>
<p class='govuk-body'>This fails WCAG 2.2 success criterion 4.1.2 Name. Role, Value.</p>

<p class='govuk-body'>We’re waiting on third parties to assist with remediation so a fix date cannot be provided.</p>

<h2 class='govuk-heading-m'>Disproportionate burden</h2>

<p class='govuk-body'>We have determined that there is a disproportionate burden to meet all the requirements for the following reasons:

<h3 class='govuk-heading-s'>Interactive tools</h3>

<p class='govuk-body'>The upload file function is unavailable whilst navigating using speech recognition software. This is a known flaw of speech recognition software.</p>

> <p class='govuk-body'>We’ve assessed the cost of fixing the navigation issues and accessing information, as well as the costs associated with interactive tools and transactions. We believe that proceeding with these changes now would place a <a class='govuk-link' href="https://www.legislation.gov.uk/uksi/2018/952/regulation/7/made" target="_blank">disproportionate burden (opens in a new tab)</a> within the meaning of the accessibility regulations. We will conduct another assessment when the supplier contract is up for renewal, likely in 2026.</p>

Reviewer: Add the `rel="noreferrer noopener"` as suggested by the [design system](https://design-system.service.gov.uk/styles/links/#external-links).

```suggestion
<p class='govuk-body'>We’ve assessed the cost of fixing the navigation issues and accessing information, as well as the costs associated with interactive tools and transactions. We believe that proceeding with these changes now would place a <a class='govuk-link' href="https://www.legislation.gov.uk/uksi/2018/952/regulation/7/made" rel="noreferrer noopener" target="_blank">disproportionate burden (opens in new tab)</a> within the meaning of the accessibility regulations. We will conduct another assessment when the supplier contract is up for renewal, likely in 2026.</p>
```

<h2 class='govuk-heading-m'>What we’re doing to improve accessibility</h2>
<p class='govuk-body'>We have a backlog of tickets created for ongoing accessibility improvements. We conduct regular user testing and audits. We will continue to use the accessibility manual to enhance knowledge within the team and resolve accessibility issues when identified.</p>

<h2 class='govuk-heading-m'>Preparation of this accessibility statement</h2>

<p class='govuk-body'>This statement was prepared on 29/02/2024. It was last reviewed on 07/05/2025.</p> 

> <p class='govuk-body'>This website was last tested on 29/02/2024.The test was carried out by Zoonou.</p>

Reviewer 2: Date should be gov uk style.

Is there a need to state who carried out the testing? It feels likes we should state that it is external party.

Reviewer: Plus 1 for the need for a GOV.UK date, for example, "29 February 2025".

Author: @defong The name of the supplier who conducted the testing was mentioned previously and is included on the accessibility template. Good spot with the date styling.
